### PR TITLE
test: fix NFTDescriptor fork test

### PR DESCRIPTION
**Pull request by smol-ninja**
Closes https://github.com/sablier-labs/v2-core/issues/893.

I have also made `NFTDescriptor_Fork_Test` a standalone contract as it doesn't really need forked asset to work.

### CODE REVIEW

**Review by PaulRBerg — APPROVED** [review]
Great idea to individualize the NFT descriptor tests

File: test/fork/NFTDescriptor.t.sol [hunk 1]
```diff
@@ -45,71 +52,107 @@ abstract contract NFTDescriptor_Fork_Test is Fork_Test {
...
         resetPrank({ msgSender: lockupDynamic.admin() });
         lockupDynamic.setNFTDescriptor(nftDescriptor);
 
-        string memory tokenURIAfter = lockupDynamic.tokenURI(streamId);
+        // Expects a call to {ISablierV2LockupDynamic.tokenURI}.
+        vm.expectCall({
+            callee: address(lockupDynamic),
+            data: abi.encodeCall(lockupDynamic.tokenURI, (streamId)),
+            count: 1
+        });
```
Comment (PaulRBerg) [inline test/fork/NFTDescriptor.t.sol:101]: It would be more helpful to expect a call to the NFT descriptor contract.

Comment (PaulRBerg) [reply test/fork/NFTDescriptor.t.sol]: Pushed a commit to address, if it looks good to you @smol-ninja we can merge

Comment (smol-ninja) [reply test/fork/NFTDescriptor.t.sol]: Yes. Thanks for the suggestion. Looks good now.

**Pull request merged**